config/application.rb

```diff
@@ -2,20 +2,7 @@
 
 require_relative "boot"
 
-require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
-require "action_view/railtie"
-require "action_cable/engine"
-# require "sprockets/railtie"
```

Comment:
:+1: 
Sprockets are not a part of `rails/all` anymore https://github.com/rails/rails/blob/main/railties/lib/rails/all.rb